msteams-platform/resources/samesite-cookie-update.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: SameSite cookie attribute 
 author: laujan
-description: Learn about Types of cookies, including SameSite cookies, their attributes, their implications in Teams tabs, task modules, and message extensions, and their authentication in Teams
+description: Learn about Types of cookies, including SameSite cookies, their attributes, their implications in Teams tabs, task modules, dialogs, and message extensions, and their authentication in Teams
```

Comment:
Instead of listing both task modules and dialogs, how about:

dialogs (task modules)

Reply:
Updated